<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="centerWidget" native="true"/>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0" rowspan="2">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
	background: rgb(25, 38, 58);
	color: rgb(255, 255, 255);
}
QComboBox::drop-down
{
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 15px;
     color: white;
     border-left-width: 0px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
     padding-left: 10px;
 }</string>
        </property>
        <property name="currentText">
         <string>step100</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>step1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>step25</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>step50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>step100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>step150</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider {
	background-color: rgb(25, 38, 58);
	border: 8px solid rgb(25, 38, 58);
}
QSlider::sub-page:horizontal {
	background-color: rgb(37, 168, 198);
	height: 2px;width:4px;
}
QSlider::add-page:horizontal {
	background-color: rgb(87, 97, 106);
	height: 2px;width:4px;
}
QSlider::groove:horizontal {
    background:transparent;  
    height:4px;
}
QSlider::handle:horizontal {
    height: 10px;
    width: 10px ;
    margin: -3 0px;
}
</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>19</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
	background-color: rgb(25, 38, 58);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="statusLabel">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>19</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
	background-color: rgb(25, 38, 58);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_video"/>
    <addaction name="actionExport_labeled_frames"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(146, 77, 194);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <action name="actionOpen_video">
   <property name="text">
    <string>Open video</string>
   </property>
  </action>
  <action name="actionExport_labeled_frames">
   <property name="text">
    <string>Export labeled frames</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
